include/nbl/ui/CWindowManagerWin32.h

```diff
@@ -19,8 +19,18 @@ namespace nbl::ui
 	{
 	public:
 		CWindowManagerWin32() = default;
-		~CWindowManagerWin32() = default;
+		~CWindowManagerWin32() {};
 
+		IWindowWin32::native_handle_t createNativeWindow(int _x, int _y, uint32_t _w, uint32_t _h, IWindow::E_CREATE_FLAGS _flags, const std::string_view& caption)
```

Comment:
dont make it public,
IWindowManager interface should creater and return `IWindow` and each derived class should implement it